src/components/Card.astro

```diff
@@ -0,0 +1,31 @@
+---
+// Properties
+type Props = {
+  description: string;
+  imgAlt: string;
+  imgSrc: string;
+  link: string;
+  title: string;
+  variant?: "default" | "padded";
+};
+const {
+  description,
+  imgAlt,
+  imgSrc,
+  link,
+  title,
+  variant = "default",
+} = Astro.props satisfies Props;
+---
+
+<article class={`card ${variant === "padded" ? "card--padded" : ""}`}>
```

Comment:
Then on this line, you could just have `class={cssClass}`

Reply:
I agree with default value as an empty string but I'm not sure if I follow this one.
We don't need a prop `class` or `cssClass` that might not be used - if that's going to be needed we can extend the component for that case (YAGNI). I'd much rather have a predefined variant that will control the look/functionality. Therefore even with default changed to empty string this logic would remain for now.
`<article class={`card ${variant === "padded" ? "card--padded" : ""}`}>`

In case of having more variants in the future I'd go for doing something like this:
```
type Props = {
  description: string;
  imgAlt: string;
  imgSrc: string;
  link: string;
  title: string;
  variant?: "" | "padded" | "outlined";
};

const variantClasses = {
  default: "",
  padded: "card--padded",
  outlined: "card--outlined",
};

const variantClass = variantClasses[variant];
---

<article class={`card ${variantClass}`}>
```

Reply:
Let me send a commit to help explain what I mean...

Reply:
Btw come to think of it we don't even need a default or empty string for it. It can be simply 
`variant?: "padded";`
for now since variant isn't a required prop. When it's used it'll be only for `padded` or any other varian in the future